README.md

```diff
@@ -33,12 +32,18 @@ Configure HTTP client (usually that is `config/common/sentry.php`):
 
 declare(strict_types=1);
 
+use GuzzleHttp\Client as GuzzleClient;
+use Http\Adapter\Guzzle7\Client as GuzzleClientAdapter;
+use Http\Client\HttpAsyncClient;
+use Http\Client\HttpClient;
+use Yiisoft\Definitions\Reference;
+
 return [
-    \Http\Client\HttpClient::class => \GuzzleHttp\Client::class,
-    \Http\Client\HttpAsyncClient::class => [
-        'class' => \Http\Adapter\Guzzle7\Client::class,
+    HttpClient::class => GuzzleClient::class,
+    HttpAsyncClient::class => [
+        'class' => GuzzleClientAdapter::class,
         '__construct()' => [
-            \Yiisoft\Factory\Definition\Reference::to(\GuzzleHttp\Client::class),
+            Reference::to(GuzzleClient::class),
```

Comment:
Can't it be `HttpClient::class`? And if so, this will be assumed without explicit configuration.

Reply:
The client accepts an object, string class name is not possible here afaik

Reply:
OK but why `Reference::to(GuzzleClient::class)` instead of `Reference::to(HttpClient::class)`?

Reply:
Do you mean it should be like that even before these changes?

Reply:
HttpClient::class

Reply:
OK. Fixed here and demo too.